details.html

```diff
@@ -0,0 +1,196 @@
+                    </div>
+                    <div class="item">
+                      <img
+                        src="./assets/details-assets/clipboard.svg"
+                        alt="Posted At Icon"
+                      />
+                      <span class="postedAt"></span>
+                    </div>
+                    <div class="item">
+                      <img
+                        src="./assets/details-assets/money.svg"
+                        alt="Salary Icon"
+                      />
+                      <span class="salary"></span>
+                    </div>
+                  </div>
+                  <button class="applyBtn">Aplica</button>
+                </div>
+              </div>
+            </div>
+
+            <div class="jd__body">
+              <div class="description">
+                <h2 class="description__header">Descrierea meseriei</h2>
+                <p id="jd"></p>
+              </div>
+
+              <div class="responsabilities">
+                <h2 class="description__header under">Responsabilitati</h2>
+                <ul class="list" id="responsabilities"></ul>
+              </div>
+              <div class="demands">
+                <h2 class="description__header under">Cerinte</h2>
+                <ul class="list" id="demands"></ul>
+              </div>
+              <div class="offerings">
+                <h2 class="description__header under">Ce oferim</h2>
+                <ul class="list" id="offerings"></ul>
+              </div>
+              <div class="general__info">
+                <h2 class="description__header under">
+                  Alte informatii (optional)
+                </h2>
+                <ul class="list" id="generalInfo"></ul>
+              </div>
+              <hr />
+              <div class="about">
+                <div class="contact">
+                  <p class="contact__name under">Persoana de contact</p>
+                  <img id="recruiter-photo" src="" alt="Recruiter" />
+                  <div>
+                    <p class="contact__person"></p>
+                    <p class="contact__role"></p>
+                  </div>
+                </div>
+                <div class="general__info">
+                  <p class="contact__name under">Date de contact</p>
+                  <ul class="list" id="contact__list">
+                    <li id="phone">Telefon:</li>
+                    <li id="fax">Fax:</li>
+                    <li id="email">Email:</li>
+                    <li id="address">Adresa:</li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+            <div class="jd__apply">
+              <button class="applyBtn">Aplica</button>
+            </div>
+          </div>
+        </main>
+      </div>
+    </div>
+    <footer class="details-footer-container">
+      <hr class="details-footer-line" />
+      <div class="details-footer-bottom">
+        <p>© 2024</p>

```

Comment:
We should add a dynamic value for the year. This needs to be changed in the index.html too.